### DISCUSSION TITLE
Module '"inquirer"' has no exported member 'Inquirer'

### DISCUSSION THREAD

**Discussion by amine0909** [jmcdo29/nest-commander#835] [Q&A]
I have an old nestjs project, so i had to upgrade it to version 9.3.9 and upgrade the nest-commander to the latest 3.6.1, but  i got this error

![image](https://user-images.githubusercontent.com/27707941/225153482-f9909454-f0d1-4735-8ffb-4891186d92fd.png)

**Comment by trm217** [comment]
Experienced the same issue. @jmcdo29 I am experiencing this with every new nest app that I setup after I install nest-commander. Any chance you could give us some insights of required Inquirer versions?

**Reply by trm217** [reply]
Never mind I found the issue, `@types/inquirer` has a non fix version, all the while inquirers version is fixed. if you manually install @types/inquirer@8.1.3` everything works fine.
Perhaps this should be updated in the package.json?

**Reply by jmcdo29** [reply]
Oh good to know. Yeah I'll try to push out a patch for that
